numpy_tasks.ipynb

```diff
@@ -9,6 +9,25 @@
     "https://docs.google.com/spreadsheets/d/1v1_X6zRlwRVjULZSrRvBYAG_raUPjTI0Ceruwh6vNXw/edit#gid=2019320840\n"
```

Comment:
<ol><li>simplify generator to one-liner</li><li>the resulting array creation can be performed in a single call</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bestetc/May/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>